index.js

```diff
@@ -8,8 +8,9 @@ function pixelmatch(img1, img2, output, width, height, options) {
 
     var threshold = options.threshold === undefined ? 0.1 : options.threshold;
 
-    // maximum acceptable square YUV distance between two colors
-    var maxDelta = 255 * 255 * 3 * threshold * threshold,
+    // maximum acceptable square distance between two colors;
+    // 35215 is the maximum possible value for the YIQ difference metric
+    var maxDelta = 35215 * threshold * threshold,
```

Comment:
I get following results (https://github.com/hastebrot/notebooks/blob/master/ipython/yiq-color-range.ipynb):

~~~
minmax(y): 0.0 255.00000255
minmax(i): -151.97438745 151.97438745
minmax(q): -133.26736305 133.26736305
~~~

~~~
dist(min, max): 272.7322528519619
dist(red, cyan): 187.6559323748062
~~~

>AFAIR it's also the same value you get for squared distance between [255, 0, 0] and [0, 255, 255].

Yes, red and cyan have the largest color distance. But, I used the weighted euclidian distance from `colorDelta()` with red and cyan and got different results. :confused: